system-addon/content-src/components/TopSites/TopSite.jsx

```diff
@@ -157,76 +149,25 @@ export class TopSite extends React.PureComponent {
   onMenuUpdate(showContextMenu) {
     this.setState({showContextMenu});
   }
-  onDismissButtonClick() {
-    const {link} = this.props;
-    if (link.isPinned) {
-      this.props.dispatch(ac.SendToMain({
-        type: at.TOP_SITES_UNPIN,
-        data: {site: {url: link.url}}
-      }));
-    }
-    this.props.dispatch(ac.SendToMain({
-      type: at.BLOCK_URL,
-      data: link.url
-    }));
-    this.userEvent("BLOCK");
-  }
-  onPinButtonClick() {
-    const {link, index} = this.props;
-    if (link.isPinned) {
-      this.props.dispatch(ac.SendToMain({
-        type: at.TOP_SITES_UNPIN,
-        data: {site: {url: link.url}}
-      }));
-      this.userEvent("UNPIN");
-    } else {
-      this.props.dispatch(ac.SendToMain({
-        type: at.TOP_SITES_PIN,
-        data: {site: {url: link.url}, index}
-      }));
-      this.userEvent("PIN");
-    }
-  }
-  onEditButtonClick() {
-    this.props.onEdit(this.props.index);
-  }
   render() {
     const {props} = this;
     const {link} = props;
     const isContextMenuOpen = this.state.showContextMenu && props.activeIndex === props.index;
     const title = link.label || link.hostname;
     return (<TopSiteLink {...props} onClick={this.onLinkClick} onDragEvent={this.props.onDragEvent} className={isContextMenuOpen ? "active" : ""} title={title}>
-        {!props.onEdit &&
-          <div>
-            <button className="context-menu-button icon" onClick={this.onMenuButtonClick}>
-              <span className="sr-only">{`Open context menu for ${title}`}</span>
-            </button>
-            <LinkMenu
-              dispatch={props.dispatch}
-              index={props.index}
-              onUpdate={this.onMenuUpdate}
-              options={TOP_SITES_CONTEXT_MENU_OPTIONS}
-              site={link}
-              source={TOP_SITES_SOURCE}
-              visible={isContextMenuOpen} />
-          </div>
-        }
-        {props.onEdit &&
-          <div className="edit-menu">
-            <button
-              className={`icon icon-${link.isPinned ? "unpin" : "pin"}`}
-              title={this.props.intl.formatMessage({id: `edit_topsites_${link.isPinned ? "unpin" : "pin"}_button`})}
-              onClick={this.onPinButtonClick} />
-            <button
-              className="icon icon-edit"
-              title={this.props.intl.formatMessage({id: "edit_topsites_edit_button"})}
-              onClick={this.onEditButtonClick} />
-            <button
-              className="icon icon-dismiss"
-              title={this.props.intl.formatMessage({id: "edit_topsites_dismiss_button"})}
-              onClick={this.onDismissButtonClick} />
-          </div>
-        }
+        <div>
+          <button className="context-menu-button icon" onClick={this.onMenuButtonClick}>
+            <span className="sr-only">{`Open context menu for ${title}`}</span>
```

Comment:
Hmmm.... I see this was added way back in #2547 but this should probably be a localized string… @sarracini ?